evm-tests/src/contracts/pureProxy.ts

```diff
@@ -0,0 +1,55 @@
+export const IPURE_PROXY_ADDRESS = "0x000000000000000000000000000000000000080b";
+
+export const IPureProxyABI = [
+    {
+        "inputs": [],
+        "name": "createPureProxy",
+        "outputs": [],
```

Comment:
```suggestion
        "outputs": [
            {
                "internalType": "bytes32[]",
                "name": "",
                "type": "bytes32[]"
             }
        ],
```
This should output the address/substrate-pubkey of the pure proxy created